### PR TITLE
Add varnish

### PR DESCRIPTION
**Pull request by Nazgolze**

### CODE REVIEW

File: roles/varnish/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    owner: root
+    group: root
+    mode: 0644
+  notify:
+    - Reload systemd
+    - Restart varnish
+
+- name: Configure varnish
+  template:
+    src: default.vcl.j2
```
Comment (fchavant) [inline roles/varnish/tasks/main.yml:45]: This file is not part of the PR?

**Review by fchavant — CHANGES_REQUESTED** [review]
Files seem to be missing.

File: roles/varnish/tasks/main.yml (hunk 1 above)
Comment (Nazgolze) [reply roles/varnish/tasks/main.yml:45]: I could create a default one. It's meant to be overridden
Thinking about it more, I think I want to make src path a variable to allow to specify different templates. I'll have another commit shortly

Comment (Nazgolze) [reply roles/varnish/tasks/main.yml]: done :+1:

**Review by fchavant — APPROVED** [review]
